packages/primitives-core/src/utils.ts

```diff
@@ -1,4 +1,4 @@
-export function interleave(vals /*: Array */) {
+export function interleave(vals: any[]): any[] {
```

Comment:
Maybe it's not worth it but the more correct declaration for this would be something like:
```suggestion
export function interleave(vals: [TemplateStringsArray, ...unknown[]]): Array<unknown> {
```
We might still want to replace `unknown` here with `any`, for practical reasons.